tests/unit/components/tab/Tab.spec.jsx

```diff
@@ -0,0 +1,37 @@
+import { render, screen, fireEvent } from '@testing-library/react'
+import Tab from '~/components/tab/Tab'
+
+describe('Tab', () => {
+  it('renders the label', () => {
+    const label = 'label text'
+    render(<Tab activeTab label={label} onClick={() => {}} />)
+    const tabLabel = screen.getByText(label)
```

Comment:
I recommend you move out the "label" variable and the onClick function from the test. Besides that, you may move out from the test "render" method and put in the beforeEach() constructions. These actions will help you to avoid redundant code.